### PR TITLE
Add Macedonian and Bosnian translations

### PR DESCRIPTION
**Pull request by OndraM**

### CODE REVIEW

File: src/languages/loader.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    languagesConfig[code] = configFunction(
+      { companyNames, ...translationOverrides[code] },
+      secondaryButtonMode,
+      cookieTable[code] || {},
+    );
+
+    return languagesConfig;
+  }, {} as Record<string, VanillaCookieConsent.Languages>);
+
+const languagesMap = {
```
Comment (literat) [inline src/languages/loader.ts:34]: Please, place this definition before it is used. I asked myself, why is this not exported, and after that, I noticed, that it was used by the function before. For more info there is this ESlint rule: https://eslint.org/docs/latest/rules/no-use-before-define

**Comment by literat** [comment]
I think you have a bad target branch here. It looks like this is based on the branch with the implementation of the language loader. 🤔

**Comment by literat** [comment]
> > I think you have a bad target branch here. It looks like this is based on the branch with the implementation of the language loader. 🤔
> 
> Yes, because we needed the live preview, and Netlify is set to only create it when the target branch is main - so this was a quick workaround, I will retarget and rebase it once the #327 is merged.

Ok, then my suggestion in code is meant to be applied in the previous PR. 🤷

**Review by literat — APPROVED** [review]

File: src/languages/loader.ts (hunk 1 above)
Comment (OndraM) [reply src/languages/loader.ts:34]: This rule is enabled and it don't triggers an error. Sooo, is this really an issue? :thinking:

Comment (OndraM) [reply src/languages/loader.ts:18]: :boom:

**Pull request merged**